### ISSUE TITLE
New task form can not be submitted without logging in

### ISSUE THREAD

**Issue by thepry** [davydovanton/ossboard#55]
I think we should either display it to anonymous users and allow to submit it or require user to login first and only then he can access this form.

Or at least add some text explaining why "Create" button is disabled.

**Comment by FredrikAugust** [comment]
Agreed, the current implementation is a bit weird, perhaps just redirect guests to the register/login page if they click the "New task" button?
